public/app/core/components/Branding/Branding.tsx

```diff
@@ -9,8 +9,8 @@ export interface BrandComponentProps {
   children?: JSX.Element | JSX.Element[];
 }
 
-const LoginLogo: FC<BrandComponentProps> = ({ className }) => {
-  return <img className={className} src="public/img/grafana_icon.svg" alt="Grafana" />;
+export const LoginLogo: FC<BrandComponentProps & { logo?: string }> = ({ className, logo }) => {
+  return <img className={className} src={`${logo ? logo : 'public/img/grafana_icon.svg'}`} alt="Grafana" />;
```

Comment:
LGTM 👌    Just a question, if it is a custom logo, `alt` value would still be "Grafana"?

Reply:
Good question, not sure what the best practice would be with a variable logo that comes from settings, but I think we could leave it as generic `Grafana` for now.